AccountService.js

```diff
@@ -40,4 +40,72 @@ export class AccountService {
     });
     return response.data;
   }
+
+  /**
+   * @method accounts
+   * @param {Object} config
+   * @param {string} config.url the base url
+   * @param {string} config.email the user's email
+   * @param {string} config.after an account's DID
+   * @param {number} config.limit how many accounts to return
+   * @return {Array} data
+   * @description calls on the adminr route and returns all
+   * accounts that match the email passed in.
+   * the cursor should be an account's DID
+   * and the limit is number.
+   */
+  async accounts({url = this.config.urls.base, email, after, limit}) {
+    const response = await axios.get(url, {
```

Comment:
What happens when axios throws?  Is it going to throw some crazy error thing on the browser console?  Is there already an established pattern for dealing with this? @dlongley @gannan08

Reply:
I actually copied the existing pattern and I think we do want the error throwing inside of the veres-wallet and not being potentially hidden by a try catch here. right now the accounts one in veres-wallet is caught and displayed in a q-alert.